features/step_definitions/basic_steps.rb

```diff
@@ -34,6 +34,10 @@
   end
 end
 
+Given(/^I am on the quizmaster Add questions page for "([^"]*)"$/) do |quiz|
+  quiz = Quiz.find_by(name: name)
+  visit quizmaster_add_questions_path(quiz)
+end
```

Comment:
It is used for "Then "Amber Rocks!" should have "1" correct answer" in the "Create quiz" feature